cmd/gaia/app/app.go

```diff
@@ -170,7 +177,32 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 	)
 	app.SetInitChainer(app.initChainer)
 	app.SetBeginBlocker(app.BeginBlocker)
-	app.SetAnteHandler(auth.NewAnteHandler(app.accountKeeper, app.feeCollectionKeeper))
+
+	authAnteHandler := auth.NewAnteHandler(app.accountKeeper, app.feeCollectionKeeper)
+
+	filterAnteHandler := func(ctx sdk.Context, tx sdk.Tx, simulate bool) (newCtx sdk.Context, result sdk.Result, abort bool) {
```

Comment:
Hmmm maybe I'm missing something glaringly obvious, but why must we have a "filterAnteHandler"? Why can't we just use the regular ante handler and have the `sendEnabled` enabled check? We won't need any build stuff for this. Yes, the ante handler would have to be provided a "keeper contract" that implements `GetSendEnabled`.

Reply:
Ohhh because it needs access to `x/bank` for message validation....sigh

Reply:
Because other people are using the bank module rn, and we don't want to add Hub specific functionality to the module.  We wanted to isolate this functionality change to just the gaia package.

Really, the proper way to do this is to allow for rerouting subroutes.  So the route of the bank module msgs would be:
`bank/send`
`bank/multisend`

And then in the router, we route
`bank/*` to the bank module, but `bank/multisend` to a special handler defined in the gaia folder.

However, we don't have time to do this rn, as this would involve changes to the router functionality.

Reply:
I guess we could just temporarily fork the entire `bank` module and add it as a special `bank` module in the gaia folder.  And then to enable transfers point it back to the SDK bank module.  That's actually not a bad idea.....

@cwgoes @jaekwon @zmanian

Reply:
Yeah anything to avoid involving the build pipeline would be idea. Seems trivial to do @sunnya97

Reply:
Seems okay to me

Reply:
I have all of these same concerns about using the build pipeline. I think, while messy, that forking the `bank` module may be the best way to do this. Looking forward to continuing this discussion in the sdk meeting today.

Reply:
Also this will be a very easy change to revert if we do that (just reimport the normal bank module into `app.go`)

Reply:
we don't know how messy it will be, this is done, lets go with this.

Reply:
Alternative as suggested by @sunnya97: https://github.com/cosmos/cosmos-sdk/pull/3807

Reply:
It's not messy at all. I really don't think we should rely on the build process to enforce business logic (especially since it ties into consensus).